src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -9,13 +9,34 @@
 #ifndef __CORECLR_HOST_H__
 #define __CORECLR_HOST_H__
 
+#if defined(_WIN32) && defined(_M_IX86)
+#define CALLING_CONVENTION __stdcall
```

Comment:
Should this have CORECLR prefix to make sure that it does not collide with where this is included in?

Reply:
makes sense